Automattic/wp-calypso
# Pricing page: add new header

### Changes proposed in this Pull Request

This PR ports the new [Jetpack.com](https://jetpack.com/) header to the Jetpack pricing page.

### Implementation notes

The implementation differs slightly from Jetpack.com:
- The search features is absent
- So is the Upgrade/Get started button
- Because of the above, the "More" functionality that wraps  in a menu overflowing items on small screens was not implemented either
- CSS and JS code is copied from Jetpack.com. It's not optimal but it made the implementation way faster.
- The header is not fixed in the pricing page

### Testing instructions

- Download the PR and run cloud
- Visit `/pricing`
- Check that the header is similar to the Jetpack.com header (keeping in mind the aforementioned notes)
- Check on desktop/mobile, authenticated/unauthenticated
- In a different locale than English, check that links bring you to Jetpack.com in the correct language

### Screenshots
<img width="1294" alt="Screen Shot 2022-02-15 at 2 08 20 PM" src="https://user-images.githubusercontent.com/1620183/154131545-3686d244-24e8-4e71-9464-9e8256471d75.png">
<img width="1281" alt="Screen Shot 2022-02-15 at 2 08 27 PM" src="https://user-images.githubusercontent.com/1620183/154131552-fabdaebc-7741-4c4a-bc08-7b42144c9f8a.png">
